dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -101,17 +101,17 @@ public void handleIndexUpdate(Entry entry, StateManagerMode command) {
         String json;
         json = getDocumentValueFromEntry(entry);
         try (RestClient restClient = RestClient.builder(new HttpHost(hostname, port, "http")).build()) {
-            String entryType = entry instanceof Tool ? "tool" : "workflow";
+            String entryType = entry instanceof Tool ? "tools" : "workflows";
```

Comment:
In line 88 if the entry is a service it would also get ignored after being set to null. i think its ok for now since we aren't indexing services. i think in the future we will be indexing them, so these functions would change to account for them